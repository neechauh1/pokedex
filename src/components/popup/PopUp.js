import React from 'react';
import './popUp.css';

const PopUp = ({ data, setIsPopUpVisible }) => {
	return (
		<div className='popup-container'>
			<div className='popup-text'>
				<span>{data}</span>
			</div>
			<div className='popup-close-icon'>
				<img
					src='./images/simple-cross.svg'
					alt='cross-icon'
					onClick={() => setIsPopUpVisible(false)}
				/>
			</div>
		</div>
	);
};

export default PopUp;
