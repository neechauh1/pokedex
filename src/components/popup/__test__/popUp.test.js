import React from "react";
import renderer from "react-test-renderer";
import PopUp from "../PopUp";

it("Image should render correctly", () => {
  const tree = renderer
    .create(<PopUp source={{ uri: "fake-image" }} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
