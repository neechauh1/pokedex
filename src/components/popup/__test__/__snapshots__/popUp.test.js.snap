// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image should render correctly 1`] = `
<div
  className="popup-container"
>
  <div
    className="popup-text"
  >
    <span />
  </div>
  <div
    className="popup-close-icon"
  >
    <img
      alt="cross-icon"
      onClick={[Function]}
      src="./images/simple-cross.svg"
    />
  </div>
</div>
`;
